packages/contracts/contracts/strategies/HackMoneyStrategy.sol

```diff
@@ -328,6 +333,7 @@ contract HackMoneyStrategy is HackMoneyStrategyBase, IHackMoneyStrategy {
         // update active strikes
         _addActiveStrike(strike.id, result.positionId);
 
+        // TODO: Remove this check as well?
```

Comment:
I think this is redundant if we specify the minExpectedPremium inside the openPosition, the OptionMarket does _checkCostInBounds